packages/meditrak-server/src/routes/importEntities/getOrCreateParentEntity.js

```diff
@@ -140,5 +140,5 @@ export async function getOrCreateParentEntity(
   // no explicit parent code provided, use either subdistrict or district as parent entity
   if (subDistrict) return { parentGeographicalArea: subDistrict, parentEntity: subDistrictEntity };
   if (district) return { parentGeographicalArea: district, parentEntity: districtEntity };
-  throw new Error('No parent entity defined');
```

Comment:
Probably part of a wider conversation but thinking it'd be good if we were able to throw a warning so that if a parent is missed by accident rather than by design the importer is notified.

Reply:
Yeah it's a good point, but I am thinking we're probably moving towards a two phase import where you import the relations separately anyway

Reply:
Removing parent_code from geojson import file results in
![image](https://user-images.githubusercontent.com/62525113/130373319-75fd33a9-e1e9-4da6-8169-a14084f0a5cc.png)

Reply:
You're taking this over as a starting point right @nick-beyondessential?

Reply:
Yes sounds good.